swift/apple/FirezoneKit/Sources/FirezoneKit/Models/DisplayableResources.swift

```diff
@@ -46,7 +46,7 @@ extension DisplayableResources {
   public func toData() -> Data? {
     ("\(versionString),"
       + (orderedResources.flatMap { [$0.name, $0.location] })
-      .map { $0.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed) }.compactMap { $0 }
+      .map { $0.addingPercentEncoding(withAllowedCharacters: .alphanumerics) }.compactMap { $0 }
```

Comment:
Will this change how Resources are displayed in the UI, or is this simply a backend change that won't affect what the user sees?

Reply:
It doesn't change how it's displayed in the UI.

We use percent encoding when serializing the resources list to data. We remove the percent encoding when we read the data to display it in the UI.